app/Http/Controllers/BookController.php

```diff
@@ -29,12 +29,21 @@ public function getSingleBook(int $id)
        ]);
     }
 
-    public function getAllBooks()
+    public function getAllBooks(Request $request)
     {
-        $books = $this->book->with('genre:id,name')
-            ->get()
-            ->makeHidden(['created_at', 'updated_at', 'blurb', 'claimed_by_name', 'page_count', 'year', 'genre_id', 'reviews_id']);
+        $request->validate([
+            'claimed' => 'boolean'
+        ]);
+
+        $books = $this->book->with('genre:id,name');
+
+        if($request->claimed == 1){
```

Comment:
Really you should write a few tests for this - make a few claimed books, and a few unclaimed books, and make sure you get the right number back with some filtering applied